custom-properties/set_org_custom_properties.py

```diff
@@ -0,0 +1,152 @@
+import os
+import requests
+import json
+import logging
+
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+
+# GitHub API base URL
+API_BASE = "https://api.github.com"
+
+# Get OAuth token from environment variable
+TOKEN = os.environ.get('OAUTH_TOKEN')
+if not TOKEN:
+    raise ValueError("OAUTH_TOKEN environment variable is not set")
+
+# Your organization name
+ORG_NAME = "hmcts-test"
+
+# Headers for API requests
+headers = {
+    "Authorization": f"Bearer {TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+def define_custom_property(org_name):
+    """
+    Define a custom property for the organization.
+    """
+    url = f"{API_BASE}/orgs/{org_name}/properties/schema/is_production"
+    data = {
+        "value_type": "true_false",
+        "required": False,
+        "default_value": "",
+        "description": "Indicates if the repository is in production",
+        "allowed_values": None,
```

Comment:
is this correct?

Reply:
This was a test, updated now. We can set it as ["true", "false"] that makes sure the only acceptable values for setting the is_production can be true or false